Jenkinsfile_CNP

```diff
@@ -91,22 +90,25 @@ withPipeline(type, product, component) {
   }
   onPR {
     enableSlackNotifications('#ccd-pr-builds')
+
     env.ENV='preview'
-    env.TEST_URL="https://ccd-data-store-api-ccd-next-hearing-date-updater-pr-${CHANGE_ID}.service.core-compute-preview.internal" // This is really not needed but here as a hack around TLS url
     env.CCD_DATA_STORE_API_BASE_URL = "https://ccd-data-store-api-ccd-next-hearing-date-updater-pr-${CHANGE_ID}.service.core-compute-preview.internal"
     env.DEFINITION_STORE_URL_BASE = "https://ccd-definition-store-ccd-next-hearing-date-updater-pr-${CHANGE_ID}.service.core-compute-preview.internal"
     env.TEST_STUB_SERVICE_BASE_URL = "http://ccd-next-hearing-date-updater-pr-${CHANGE_ID}-ccd-test-stubs-service" // NB : when def file is imported this will lead to data-store.preview -> test-stub.preview.
 
+    enableFullFunctionalTestNonServiceApp(env.CCD_DATA_STORE_API_BASE_URL)
```

Comment:
Maybe could be moved up as it is used in onPR and onNonPR?

Reply:
Should now be resolved in 27ac460b1b74344d8d5a4eaf92d082c098dae115.